src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -27,6 +34,34 @@
                  'edge': "_make_edge"
                 }
 
+class _ReusableDriver(webdriver.Remote):
+    def __init__(self, command_executor, sid, keep_alive=False, file_detector=None):
+
+        self.reuse_session_id = sid
+        super(_ReusableDriver, self).__init__(command_executor, desired_capabilities={},
+                                              browser_profile=None,
+                                              proxy=None, keep_alive=keep_alive,
+                                              file_detector=file_detector)
+
+    def stop_client(self):
+        if self.pid is not None and self.pid != '':
+            logger.info("Stopping driver for session '%s' with PID: %s" % (self.session_id, self.pid))
+            try:
+                os.kill(int(self.pid), signal.SIGTERM)
+            except Exception:
```

Comment:
1 done
2 I'd say no. The side-effect is that the affected browser just stays open and is not anymore "reachable" via Selenium and must be closed manually. IMHO this is okay when considering the use cases for these KW's = TC development i.e. not for production use.